lsmtool/tableio.py

```diff
@@ -1183,6 +1207,52 @@ def getGSM(position, radius):
     return outFile
 
 
+def getLoTSS(position, radius):
+    """
+    Returns table from a LoTSS search.
+
+    Parameters
+    ----------
+    position : list of floats
+        A list specifying a new position as [RA, Dec] in either makesourcedb
+        format (e.g., ['12:23:43.21', '+22.34.21.2']) or in degrees (e.g.,
+        [123.2312, 23.3422])
+    radius : float or str, optional
+        Radius in degrees (if float) or 'value unit' (if str; e.g.,
+        '30 arcsec') for cone search region in degrees
+
+    """
+    import tempfile
+    import subprocess
+
+    log = logging.getLogger('LSMTool.Load')
+
+    columnMapping = {'Source_Name':'name', 'RA':'ra', 'DEC':'dec', 'Total_flux':'i',
+                     'DC_Maj':'majoraxis', 'DC_Min':'minoraxis', 'PA':'orientation',
+                     'referencefrequency':1.4e8}
+
+    outFile = tempfile.NamedTemporaryFile()
+    RA = RA2Angle(position[0])[0].value
+    Dec = Dec2Angle(position[1])[0].value
+    try:
+        radius = Angle(radius, unit='degree').value * 60.0  # arcmin
+    except TypeError:
+        raise ValueError('LoTSS query radius not understood.')
```

Comment:
Could be good to add the query in the error message.

Reply:
Good idea -- I added it.